### ISSUE TITLE
Train a model using frozen H&E data

### ISSUE THREAD

**Issue by dzenanz** [Kitware/BANFF-AID#21]
Resume from PAS checkpoint using a combination of our data and ComPRePS data. This will allow easier and better sanity checking. Maybe use only frozen H&E data in the validation set?

Possible experimentation:
 - switch to training using only frozen H&E
 - train from scratch (or pretrained on COCO), not using PAS data
